### ISSUE TITLE
Discussion: Thoughts on few shots?

### ISSUE THREAD

**Issue by AntiLibrary5** [tue-mps/eomt#34]
Hey. 
Interesting paper and thanks for making the code available. In the paper you mainly explore standard segmentation benchmarks. I was wondering since eomt's simplicity makes it benefit from foundation vision models like dinov2,  we should expect decent few shot performance. You finetune the mask module head but also the backbone (dinov2) in schedule. I was wondering if you explored linear probing with the backbone frozen or adapter (low rank) layers (faster training in addition to the fast inference) as finetuning with the backbone might lead to overfitting with few images (in case you even tried). 
PS: nothing to criticize the work. just discussion since you know the ins and outs more of the model/work/code.

Thanks.

**Comment by NiccoloCavagnero** [comment]
Closing the issue as the discussion stalled.
